contribs/gmf/examples/style/mobileapp-vars.less

```diff
@@ -1,8 +1,13 @@
+@app-margin: 13px;
```

Comment:
What I mean is that the geolocation button should be treated as a map control. Maybe an extension of `ol.control`.
This would prevent errors if the OpenLayers CSS classes change.
An other option is to get rid of [+] and [-] buttons and manage them directly in our application.